### ISSUE TITLE
[Programmatic Access - Make code Arcade- Projects]: Main landmark is not defined for the 'Projects' page.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#6100]
### User Experience:
Visually impaired people who depend on screen reader will get impacted if main landmark is not defined on the page. Due to which the end users will not be able to know the main context of the page and will find difficulty in understanding the page structure.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: [https://arcade.makecode.com/.](https://arcade.makecode.com/projects).
2. Projects page will open.
3. Turn on the AI (Accessibility Insights) and observe whether main landmark is defined for the page or not.

### Actual Result:
**Issue:** Main landmark is not defined for the 'Projects' page.

### Expected Result:

- Main landmark should be defined for the 'Projects' page to know the main context the page and to understand the page structure.
- All the content present on the page should be enclosed within the landmarks.

### MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
[[aria-input-field-name](https://accessibilityinsights.io/info-examples/web/aria-input-field-name)
](https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/HTML5.html)

### UIA/DOM details available:
Yes (Refer the attachment)

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:
**OS Version:** Windows 11 24H2 (OS Build:26100.3037)
**Browser:** Chrome Version 132.0.6834.196 (Official Build) (64-bit)
**Test Matrix:** Chrome + NVDA
**URL:** https://arcade.makecode.com/projects

### Attachment:

<img width="980" alt="Image" src="https://github.com/user-attachments/assets/919d2fd0-b681-41cd-be4f-b481c6532806" />

**Comment by vroopea** [comment]
#Accessibility; #A11yMAS; #A11ySev3; #MAS1.3.1; #AILmited; #HCL; #Benchmark; #Win11-Chrome; #BM-HCL-MakeCode-Feb2025

**Comment by srietkerk** [comment]
This should be fixed with the latest arcade release. 

![Image](https://github.com/user-attachments/assets/d68c9c4b-abcd-43ed-871c-f2999fe315b7)

**Comment by Yamini234u** [comment]
As checked issue on https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug

**Comment by Yamini234u** [comment]
As checked issue on https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug

**Comment by Yamini234u** [comment]
As checked issue on https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug

**Comment by Yamini234u** [comment]
As checked issue on https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug
